src/org.freedesktop.fwupd.xml

```diff
@@ -826,6 +826,35 @@
       </arg>
     </method>
 
+    <!--***********************************************************-->
+    <method name='SecurityHarden'>
+      <doc:doc>
+        <doc:description>
+          <doc:para>
+            Security hardening for HSI.
+          </doc:para>
+        </doc:description>
+      </doc:doc>
+      <arg type='s' name='appstream_id' direction='in'>
+        <doc:doc>
+          <doc:summary>
+            <doc:para>
+              The fwupd AppStream ID, e.g. 'org.fwupd.hsi.Kernel.Lockdown'.
+            </doc:para>
+          </doc:summary>
+        </doc:doc>
+      </arg>
+      <arg type='s' name='action' direction='in'>
```

Comment:
if you follow my suggestion I think this can be converted into a boolean or integer and then it's one less string to compare and pass around a bunch.

Reply:
I actually covert it to enum before but I hit a file "#include" issue when CI built the fu-util.c that forced me to covert the action in the daemon.
If it could be a boolean I'm happy to do this :)

Reply:
It is ok when building code at the local. There is an issue (../src/src/fu-engine.h not found when compiling fu-utils.c) when running CI for Fedora package) . fu-engine-security.h includes fu-engine.c and fu-utils.c includes fu-engine-security.h for the enum declaration. The issue appears :(